app/serializers/api/state_serializer.rb

```diff
@@ -1,7 +1,12 @@
 class Api::StateSerializer < ActiveModel::Serializer
-  attributes :id, :name, :abbr
+  attributes :id, :name, :abbr, :label
 
   def abbr
     object.abbr.upcase
   end
+
+  def label 
+	eval Spree::Config[:state_display]
```

Comment:
this should be `public_send` instead. That is the way to execute dynamic method calls in ruby.